channels/serializers.py

```diff
@@ -262,6 +263,12 @@ def update(self, instance, validated_data):
             elif removed is False:
                 api.approve_post(post_id)
 
+        if "ignore_reports" in validated_data:
+            ignore_reports = validated_data["ignore_reports"]
+            if ignore_reports is True:
```

Comment:
Is there the ability to stop ignoring reports if this is false?

Reply:
There is in `praw`, but there's no requirement from a product standpoint at the moment.